src/components/Buttons/Button/__tests__/__snapshots__/Button.spec.tsx.snap

```diff
@@ -3,21 +3,42 @@
 exports[`<Button> that renders a button should add classes when props are changed 1`] = `
 <div>
   <button
-    class="Button Button--isPrimary Button--isBlock Button--isLoading Button--size_large"
+    class="Button Button--isBlock Button--size_large Button--variant_filled Button--context_primary"
+    disabled=""
```

Comment:
wouldn't we expect this to be `true` for a loading button?

Reply:
As I understand it, in HTML, boolean attributes are considered truthy if the value is an empty string. So this is indeed true. If it isn't there, it is false.